Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerWidget.cpp

```diff
@@ -918,6 +919,13 @@ namespace AzToolsFramework
 
     void EntityOutlinerWidget::OnEntityPickModeStarted()
     {
+        // If we're in component mode, it's possible the outliner isn't currently enabled. If so,
+        // make sure to enable it while in pick mode.
+        if (!isEnabled())
```

Comment:
suggestion: Same as before, might be worth adding an explicit check or assert we're definitely in Component Mode as this hasn't been disabled for some other reason